### PR TITLE
Update schedule command

### PR DESCRIPTION
**Pull request by t-cheepeng**
- Schedule now pretty printed that is more human readable.
- Schedule command will not accept dates before the current date now.

### CODE REVIEW

File: src/main/java/seedu/exercise/model/property/Date.java [hunk 1]
```diff
@@ -119,26 +126,64 @@ public static Date getOneWeekBeforeToday() {
...
         try {
-            sDate = LocalDate.parse(startDate.toString(), formatter);
-            eDate = LocalDate.parse(endDate.toString(), formatter);
+            sDate = LocalDate.parse(startDate.toString(), STANDARD_DATE_TIME_FORMATTER);
+            eDate = LocalDate.parse(endDate.toString(), STANDARD_DATE_TIME_FORMATTER);
         } catch (DateTimeParseException e) {
             return new ArrayList<>();
         }
-        days = (int) DAYS.between(sDate, eDate) + 1;
+        days = (int) ChronoUnit.DAYS.between(sDate, eDate) + 1;
```
Comment (garylyp) [inline src/main/java/seedu/exercise/model/property/Date.java]: What does ChronoUnit do?

File: src/test/java/seedu/exercise/model/property/DateTest.java [hunk 2]
```diff
@@ -54,6 +58,11 @@ public void isEndDateAfterStartDate() {
         assertFalse(Date.isEndDateAfterStartDate("02/01/2019", "01/11/2018"));
     }
 
+    @Test
+    public void isEndDateAfterStartDate_invalidDateString_returnsFalse() {
+        assertFalse(Date.isEndDateAfterStartDate(CommonTestData.INVALID_DATE_ALPHABETS, CommonTestData.VALID_DATE));
```
Comment (garylyp) [inline src/test/java/seedu/exercise/model/property/DateTest.java]: Won't it throw ParseException if date string is invalid?

File: src/main/java/seedu/exercise/model/property/Date.java (hunk 1 above)
Comment (t-cheepeng) [reply src/main/java/seedu/exercise/model/property/Date.java]: Its a set of standard units for dates. Like century, month, date, year,etc... Represented as an enum inside the java class.

Read the javadocs to find more

Comment (garylyp) [reply src/main/java/seedu/exercise/model/property/Date.java]: Noted

File: src/test/java/seedu/exercise/model/property/DateTest.java (hunk 2 above)
Comment (garylyp) [reply src/test/java/seedu/exercise/model/property/DateTest.java]: Okay

**Pull request merged**